### PR TITLE
Add weighted averages

### PR DESCRIPTION
**Pull request by ZIMkaRU**
This PR adds ability to fetch weighted averages data from the BFX API using `v2/auth/r/trades/calc` endpoint

---

Basic changes:
- makes `symbol` param required for weighted averages
- allows only one `symbol` with params schema
- reworks `WeightedAveragesReport` service to be able to work with api data
- changes test cases for using one symbol

---

Notes:
- request/response data structures are the same as it was before
- If `nextPage === true` it means that data is not consistent and need to change timeframe. That is, when requesting, we set a limit of 100k and in the api response it returns the number of trade records participating in the calculation, if their number has reached the requested limit, this means that the user may not see the data for the desired period of time and it needs to be reduced. This message should be displayed to the user

---

**Depends** on these PRs:
- https://github.com/bitfinexcom/bfx-api-node-models/pull/82
- https://github.com/bitfinexcom/bfx-api-mock-srv/pull/54
- https://github.com/bitfinexcom/bfx-api-node-rest/pull/107

### CODE REVIEW

**Review by ezewer — COMMENTED** [review]
small comment

File: workers/loc.api/weighted.averages.report/index.js [hunk 1]
```diff
@@ -50,6 +63,69 @@ class WeightedAveragesReport {
     }
   }
 
+  async _getWeightedAveragesReportFromApi (args) {
+    const limit = 100_000
```
[the remark this answers is not in the document]
Comment (ZIMkaRU) [reply workers/loc.api/weighted.averages.report/index.js:67]: JS shipped a new feature called `numeric separator` to improve readability on numbers. The underscore `_` was introduced in ES2021 as a separator in numbers and we can use it for decimal, binary, hexadecimal, and BigInt numbers

[the remark this answers is not in the document]
Comment (ezewer) [reply workers/loc.api/weighted.averages.report/index.js:67]: interesting

**Review by ezewer — APPROVED** [review]
lgtm!

**Pull request merged**